data/map systems.txt

```diff
@@ -26144,7 +26199,6 @@ system Polaris
 	fleet "Small Syndicate" 1000
 	fleet "Large Syndicate" 2000
 	fleet "Small Core Pirates" 1500
-	fleet "Korath Large Raid" 50000
```

Comment:
Why is this no longer being raided?

Reply:
I don't remember, looks like a mistake to me 😅

Reply:
Korath raiding major Syndicate systems causes a lore problem: Korath cannot be secret if they're raiding populated Syndicate worlds. This story and its associated lore establishes the bounds of what they actually raid, and why. Any benefits Korath get from Syndicate space comes remotely through New Tortuga's illicit activities. (Such as jobs board jobs.)